taxi_service/settings.py

```diff
@@ -140,3 +142,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The 'CRISPY_TEMPLATE_PACK' is correctly set to 'bootstrap4'. This is necessary for using crispy forms with the specified template pack as required by the task description.